.travis.yml

```diff
@@ -6,10 +6,13 @@ language: python
 python:
   - "2.7"
 
+addons:
+  postgresql: "9.4"
+
 cache:
   directories:
     - $HOME/.cache/pip
-    - $HOME/caffe
+    #- $HOME/caffe
```

Comment:
To run some unit tests for Caffe from the latest PR on unit tests we need opencv, so we need to removed the old cached version from the master branch and then add it back to cache